src/common/transformations/include/transformations/dimension_tracking.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Why not in the `src/common/transformations/include/transformations/common_optimizations/dimension_tracking.hpp` as source file?